source/versions/filenames.rst

```diff
@@ -0,0 +1,117 @@
+    use more recent versions of Forge, than the one used for SpongeForge, but we can't always guarantee compatibility.
+
+.. warning::
+
+    When investigating crash issues, you can freely try newer versions of Forge than listed on the SpongeForge Jar.
+    However, it is recommended to also check with the matching version, to make sure your issue is not related to a
+    version mismatch.
+    Even though there will be no guarantee of compatibility, please report any breakage to the issue tracker, so that
+    SpongeForge can be updated.
+
+SpongeVanilla
+=============
+
+The information for SpongeVanilla is identical to SpongeForge except that SpongeVanilla does not contain a Forge build 
+number. So, the above examples of SpongeForge jar files will look like the following for SpongeVanilla:
+
+``spongevanilla-1.12.2-7.1.4.jar`` and ``spongevanilla-1.12.2-7.1.5-RC3449.jar``
+
+SpongeAPI
+=========
+
+The format of the filename is:
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>.0-shaded.jar``
+
+**OR**
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>.0-<Date>.<Time>-<BuildNumber>-shaded.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``APIMajor``         | The major version of SpongeAPI implemented by this file (the ``X`` in                         |
+|                      | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``LatestAPI``        | The minor version of SpongeAPI implemented by this file (the ``Y`` in                         |
+| ``MinorRelease``     | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Date``             | The date when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Time``             | The time when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``<BuildNumber>``    | The build number of Sponge. Supply this number when reporting bugs or seeking support.        |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+.. note::
+
+    The first format without the build information is the release and recommended build format. The second format with 
+    build information is the *SNAPSHOT* format. This version is still in development.
```

Comment:
Maybe mention that this format is only used as raw downloads. Depending on the API via Gradle or Maven will download a formatted named jar as the recommended build format (`spongeapi-7.2.0-SNAPSHOT.jar`). Downloading via browser for specific builds will result in the timestamped jar file.

Reply:
`spongeapi-7.2.0-SNAPSHOT.jar` is the recommended build format? I thought the `-SNAPSHOT` label means it is still in development.

Reply:
Added another format. May need to change further to have the appropriate label.